test/integration/preflight/preflight_test.go

```diff
@@ -49,20 +49,20 @@ func (p PlaybookTest) Run(t *testing.T) {
 	b, err := cmd.CombinedOutput()
 	if p.ExitCode == 0 {
 		if err != nil {
-			t.Errorf("playbook execution failed: %v", err)
+			t.Errorf("playbook %s execution failed: %v", p.Path, err)
```

Comment:
Rebase against https://github.com/rhcarvalho/openshift-ansible/commits/pre-flight-checks, we've got better errors now (and colors, if that's of any use). :rainbow: